Sources/PluginViewController/PluginViewController.swift

```diff
@@ -87,6 +124,23 @@ final class PluginViewController: UIViewController {
         closeEventHandler?()
     }
 
+    @objc private func keyboardDidShow(_ notification: Notification) {
+        guard let userInfo = notification.userInfo else {
+            return
+        }
+        guard let endFrame = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect else {
+            return
+        }
+
+        tableView.contentInset.bottom = endFrame.height
+        tableView.scrollIndicatorInsets = tableView.contentInset
+    }
+
+    @objc private func keyboardWillHide(_ notification: Notification) {
+        tableView.contentInset.bottom = 0
```

Comment:
Maybe it would be better to use `tableView.contentInset.bottom = view.safeAreaInsets.bottom` here instead of just `0`?